src/controllers/solidity/index.ts

```diff
@@ -10,6 +10,7 @@ import {
     uniqueFilePath,
 } from "../../core/utils";
 import fs from "fs";
+import mv from 'mv';
```

Comment:
Need to add mv node module to package.json

Reply:
Done